guides/Hardware-Budget-Uses.md

```diff
@@ -39,12 +39,22 @@ Please feel free to add what you have got through the YLD Hardware budget, it ca
 - [Hub USB C, ethernet, HDMI 4k, 3.5mm audio, card reader](https://www.amazon.es/gp/product/B07GKXN841/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1) - *[@renatoc](https://github.com/renatoc)*
 
 
-
 #### Stands
 
 - [Portable foldable x stand](https://www.amazon.co.uk/gp/product/B007BD16KC/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1) - *[@jgantunes](https://github.com/jgantunes)*
 
+
 #### Hardware Security Keys
 
 - [YubiKey 5 NFC](https://www.yubico.com/product/yubikey-5-nfc/#yubikey-5-nfc) - *[@jgantunes](https://github.com/jgantunes)* *[@Couto](https://github.com/Couto)*
 
+
+#### External Hard Drives
+
+- [LaCie Portable HDD](https://www.amazon.co.uk/dp/B01EIP37IY/ref=twister_B07B94XJPM?_encoding=UTF8&psc=1) - *[@joescho](https://github.com/joescho)*
```

Comment:
Please clean up the query params from this URL, don't need Joe-Tracking-Ident :)